test/conformance/kuttl/verifyImages/clusterpolicy/standard/configmap-context-lookup/README.md

```diff
@@ -0,0 +1,11 @@
+## Description
+
+The configmap context lookup uses informer's cache internally, the background processing should use the same to resolve configmap context without crashing Kyverno.
```

Comment:
Would you also make a referring note here to there being a second test for a similar behavior in the validate folder?

Reply:
Done.